main.tf

```diff
@@ -107,14 +108,14 @@ resource "google_compute_target_https_proxy" "https_proxy" {
 
 resource "google_compute_global_address" "lb_ip" {
   count        = local.create_bucket_lb ? 1 : 0
-  name         = format("lb-ip-%s", local.lb_resource_name_suffix)
+  name         = format("${local.lb_ip_name}-%s", local.lb_resource_name_suffix)
   ip_version   = "IPV4"
   address_type = "EXTERNAL"
 }
 
 resource "google_compute_global_forwarding_rule" "fw_rule" {
   count      = local.create_bucket_lb ? 1 : 0
-  name       = format("forwarding-rule-%s", local.lb_resource_name_suffix)
+  name       = format("bucket-fwrule-%s", local.lb_resource_name_suffix)
```

Comment:
Let's use the name `bucket-forwarding-rule-`

Otherwise people might get confuse fw with firewall rule.

Reply:
Good point 👍🏽

Reply:
Was meaning to keep it short. What do you think about `bucket-fwdrule-`  ?

Reply:
Keeping the longer version doesn't hurt. I prefer better readability. You can see the TF module names are quite long and elaborate. It helps you understand what exactly it is without you brain remapping fw => Firewall, fwd => Forwarding.

Reply:
"brain-remapping" - perfectly illustrating :)

Sure. Will do the changes.

Reply:
@fai555 updated